proto/src/main/proto/Rss.proto

```diff
@@ -662,4 +669,30 @@ message ReassignOnBlockSendFailureResponse {
   MutableShuffleHandleInfo handle = 3;
 }
 
+message StartSortMergeRequest {
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 partitionId = 3;
+  bytes uniqueBlocksBitmap = 4;
+}
+
+message StartSortMergeResponse {
+  StatusCode status = 1;
+  string retMsg = 2;
+}
+
+message GetSortedShuffleDataRequest {
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 partitionId = 3;
+  int64 blockId = 4;
```

Comment:
Why do we use a blockId instead of offset?

Reply:
The blocks obtained from the map side will be merged into a group of new blocks on the server side. These new blocks are counted from 1, so we can use blockid to read all the data. We only read one block at a time, and then increase the blockid by 1 for the next read.

On the other hand, getSortedShuffleData may get data from memory or file. In other APIs, we use blockid to get data from memory and offset to get data from file. In the new API getSortedShuffleData, blockid is used uniformly, it is more convenient.

Reply:
Is the format  the same as the block's? Do we need a new concept?

Reply:
It is a block, not a new concept. The original block code is reused, so flushing will be performed when necessary.
There are only two differences from the original block:
(1) This group of new blocks increases in sequence from 1, the purpose is to read sequentially.
(2) This group of new blocks is under a new appId, that is, the original appid + "@RemoteMerge". getSortedShuffleData will sequentially get the blocks with this new appid.

Reply:
So should we just rename it to the mergeBlockId?

Reply:
I think it's okay to change it or not. If change to mergeBlockId, seems more clearer, can tell all that the ID is read sequentially starting from 1. Then let’s rename it.